tests/unittest/llmapi/apps/_test_trtllm_serve_multimodal_example.py

```
@@ -61,12 +61,15 @@ def example_root():
 @pytest.mark.parametrize("exe, script",
                          [("python3", "openai_chat_client_for_multimodal.py"),
                           ("bash", "aiperf_client_for_multimodal.sh")])
-def test_trtllm_serve_examples(exe: str, script: str,
+def test_trtllm_serve_examples(exe: str, script: str, model_name: str,
                                server: RemoteOpenAIServer, example_root: str):
```

Ruff (0.14.14)

[warning] Unused function argument: `server`

(ARG001)